client/state/plugins/wporg/actions.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+const debug = require( 'debug' )( 'calypso:wporg-data:actions' );
+
+/**
+ * Internal dependencies
+ */
+import wporg from 'lib/wporg';
+import utils from 'lib/plugins/utils';
+import { RECEIVE_WPORG_PLUGIN_DATA, FETCH_WPORG_PLUGIN_DATA } from 'state/action-types';
+
+/**
+ *  Local variables;
+ */
+let _fetching = {};
+
+export default {
+	fetchPluginData: function( pluginSlug ) {
+		return ( dispatch ) => {
+			if ( _fetching[ pluginSlug ] ) {
+				return;
+			}
+			_fetching[ pluginSlug ] = true;
+
+			setTimeout( () => {
```

Comment:
I get one of those pesky "Cannot dispatch in the middle of a dispatch" without it  ... I'll do some tests to see if I can get rid of it